marathon-client/README.md

```diff
@@ -0,0 +1,88 @@
+# marathon-client [![Build Status](https://travis-ci.org/mohitsoni/marathon-client.png)](https://travis-ci.org/mohitsoni/marathon-client)
+
+This project is a Java library for communicating with Marathon API. At this point this library supports version v2 of Marathon API, please refer [docs](https://mesosphere.github.io/marathon/docs/rest-api.html) for more details.
+
+## Using marathon-client in your maven project
+
+Add marathon-client as a dependency:
+
+```
+<dependency>
+  <groupId>com.github.mohitsoni</groupId>
```

Comment:
TBH I'd like to keep marathon-client unmodified too much at least for now
Its possible I'll refactor so much of its code that I'll be able to remove original authors groups etc with clear conscience